src/style-spec/reference/v8.json

```diff
@@ -181,6 +181,11 @@
       "type": "promoteId",
       "doc": "A property to use as a feature id (for feature state). Either a property name, or an object of the form `{<sourceLayer>: <propertyName>}`. If specified as a string for a vector tile source, the same property is used across all its source layers."
     },
+    "volatile": {
+      "type": "boolean",
+      "default": false,
+      "doc": "Setting this flag to true indicates that the source tiles shall not be stored in the local cache." 
```

Comment:
```suggestion
      "doc": "A setting to determine whether a source's tiles are cached locally." 
```

Reply:
I think this language is a bit closer to how other source properties are documented.